build.gradle

```diff
@@ -101,6 +104,10 @@ dependencies {
     implementation fg.deobf("mezz.jei:jei-1.16.4:${config.jei_version}")
     compileOnly fg.deobf("com.resourcefulbees:ResourcefulBees:${config.rb_version}")
     apiCompileOnly fg.deobf("com.resourcefulbees:ResourcefulBees:${config.rb_version}")
+    compileOnly fg.deobf("slimeknights.mantle:Mantle:1.16.5-${config.mantle_version}")
+    compileOnly fg.deobf("slimeknights.tconstruct:TConstruct:1.16.5-${config.tc_version}")
```

Comment:
Not needed.

The API SourceSet will provide the dependency to the main sourceset.